db/src/db.rs

```diff
@@ -26,7 +26,7 @@ use entids;
 use mentat_tx::entities as entmod;
 use mentat_tx::entities::{Entity, OpType};
 use errors::{ErrorKind, Result, ResultExt};
-use types::{Attribute, AttributeBitFlags, DB, Entid, IdentMap, Partition, PartitionMap, Schema, TypedValue, ValueType};
+use types::{AVMap, AVPair, Attribute, AttributeBitFlags, DB, Entid, IdentMap, Partition, PartitionMap, Schema, TypedValue, TxReport, ValueType};
```

Comment:
I'm happy to push these onto multiple lines for clean diffs.  I'll do it throughout.